airflow/plugins_manager.py

```diff
@@ -256,29 +248,18 @@ def initialize_web_ui_plugins():
 
     log.debug("Initialize Web UI plugin")
 
-    admin_views = []
     flask_blueprints = []
-    menu_links = []
     flask_appbuilder_views = []
     flask_appbuilder_menu_links = []
 
     for plugin in plugins:
-        admin_views.extend(plugin.admin_views)
-        menu_links.extend(plugin.menu_links)
         flask_appbuilder_views.extend(plugin.appbuilder_views)
         flask_appbuilder_menu_links.extend(plugin.appbuilder_menu_items)
         flask_blueprints.extend([{
             'name': plugin.name,
             'blueprint': bp
         } for bp in plugin.flask_blueprints])
 
-        if (admin_views and not flask_appbuilder_views) or (menu_links and not flask_appbuilder_menu_links):
```

Comment:
```
if (plugin.admin_views and not plugin.appbuilder_views) or (plugin.menu_links and not plugin.flask_appbuilder_menu_links):
```
In my opinion, this message may remain, but it seems to me that the global variable can be deleted completely. Users who migrate to RBAC proposed by airflow upgrade-check may not be aware of the consequences.

Reply:
Done @mik-laj